.gitignore

```diff
@@ -15,3 +15,5 @@ lerna-debug.log*
 # when working with contributors
 package-lock.json
 yarn.lock
+
+.vscode
```

Comment:
This should probably be in your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) too!

Reply:
I've added it because the project explicitly instructs you to create settings for `.vscode` in the `CONTRIBUTING.md`.

Reply:
Yeah I agree we should add it. Just wanted to let you know you can use a global gitignore file for it too.